BOOTSEQ.md

```diff
@@ -4,77 +4,160 @@
 
 Entities optionally participating in the boot/reboot/shutdown processes:
 
-- Platform UEFI/BIOS of the Server
-- Platform BMC of the Server
+- Platform UEFI/BIOS of the host
+- Platform BMC of the host
 - xPU BMC
 - xPU ATF/UEFI/BIOS
-- additional xPUs that reside within the Server
+- additional xPUs that reside within the host
 
 ## Assumptions
 
-- Resetting the xPU is assumed to cause PCIe errors that will crash the server (at least for some scenarios).
-  - This will cause challenges for deploying an OS on the xPU and when running multi-xPU deployments. Perhaps OPI should work with Linux Foundation to encourage host and OS developers to properly manage the response by the host and the xPU(s) from a Reset
-  - There are different kinds of resets. Genrally XPU core complex reset and XPU PCIe MAC reset should be separated for minimal disruptive ISSU etc. but a complete SoC reset of the xPU will cause PCIe errors and needs reboot the server.
-- There are use cases for network boot only.
+- Resetting the xPU is assumed to cause PCIe errors that may impact the operation of the host OS.
+  - There are different kinds of resets. Generally XPU core complex reset and XPU PCIe MAC reset should be separated for minimal disruptive ISSU etc. but a complete SoC reset of the xPU will cause PCIe errors and may result in a controlled or uncontrolled host reset.
+  - See  _Host Reset or Crash_ and _xPU Reset or Crash_ sections for specifics.
+
+- There are use cases for network boot only where the host is used only to provide power and cooling to one or more xPUs.
```

Comment:
Is it only "bump in the wire" case? It may be a bare-metal hosting case, where host is completely isolated from the IPU (meaning, to the level PCIe card can be isolated from host - power+thermals).